src/karmabot/commands/note.py

```diff
@@ -0,0 +1,113 @@
+import datetime
+import re
+from typing import Callable, Dict, Union
+
+from karmabot.db import db_session
+from karmabot.db.karma_note import KarmaNote
+from karmabot.db.karma_user import KarmaUser
+
+NOTE_CMD_PATTERN = re.compile(r"note\s(\w+)\s?(.*)")
+NOTE_COMMANDS: Dict[str, Callable] = {
+    "add": lambda text_arg, user: _add_note(text_arg, user),

```

Comment:
do we need the `lambda`s here? can't we just do `"add": _add_note,`?

Reply:
I like the use of constants here btw

Reply:
I implemented it without lambda but now the constant has to come at the end of the file...dont like it but the alternative with lambda is uglier and another solution, putting it inside its own module, is overkill